ericduart/robot-generator
# New Feature. Save favorite robots

## Summary of changes
This PR introduces the following features:

- **Entity Framework + SQLite**  
  Implements a SQLite database using Entity Framework to save the robots that users marked as "liked".  

- **CSV Exporter**  
  Adds functionality to export favorite robots into a CSV file. 

### How to Test
1. Like a few robots in the application by clicking the star icon.
2. Use the CSV export feature (click the file icon underneath the star icon) and select a folder to save the file.
3. Verify that the exported CSV file contains the correct robot data.
